tools/configen/configen/config.py

```diff
@@ -17,7 +17,7 @@ class Flags:
 class ModuleConf:
     name: str = MISSING
     classes: List[str] = MISSING
-    default_flags: Optional[Flags] = None
+    default_flags: Optional[Flags] = Flags()
```

Comment:
I think this can be non optional (and that you still does not need to specify those in the config).

Reply:
I think you're right. Just tested it. Is this because all the field in `Flags` are optional?

Reply:
The final config is the merge of the  top level config dataclass and the config from the config file.
the dataclass is providing the default values.